CONTRIBUTING.md

```diff
@@ -1,44 +1,81 @@
-# Development guide
 
-This document is intended for developers only.
+###This document is intended for developers only.
+### Runtime dependencies
+Since setup.cfg lists these dependencies.
+```
+pip install click "ruamel.yaml<0.18" simplejson~=3.17 requests~=2.25 click~=7.1 psutil~=5.8 scipy~=1.6 coloredlogs~=15.0
+```
+
 
-## Testing
+## Writing tests
 
 Write unit tests as functions without arguments prefixed with ``_unittest_``;
 optionally, for slow test functions use the prefix ``_unittest_slow_``.
 Generally, simple test functions should be located as close as possible to the tested code,
 preferably at the end of the same Python module.
 
-The directory `tests/deps` contains various test dependencies, including `sitecustomize.py`,
-which is used to measure code coverage in subprocesses (which the test suite spawns a lot of, naturally).
+## Being able to run tests
+
+### Installable dependencies
+
+Install [Nox](https://nox.thea.codes): `pip install nox`
+
+Install ncat ([Netcat](https://nmap.org/ncat/)): `sudo apt install ncat`
+
+### Restrictions to running on your computer
 
-When running tests on GNU/Linux, ensure that the current user is allowed to use `sudo` without an
-interactive password prompt.
+When running tests on GNU/Linux, ensure either that the current user is allowed to use `sudo` without an
+interactive password prompt or when you're around, you can also enter the password when prompted.
 This is needed for setting up `vcan` interfaces, loading relevant kernel modules, and setting up packet capture.
 
-### Manual testing
+### Mypy inspections
+#### Having mypy analyze the code without running the whole test suite
 
-Some tools with rich UI are difficult to fully test manually.
-These should be validated manually; please find instructions in the corresponding files under `tests/cmd`.
+The test suite should pass and one part of it is the mypy code analysis that takes place at the end of the execution of the long test suite.
 
-### Using Nox
 
-Install [Nox](https://nox.thea.codes): `pip install nox`
+1. Be in the yakut root directory
+2. Run the long test suite once with ```nox``` 
+3. Change directory to ```.nox/test-3-8/tmp```
+4. This is one of the environments that nox creates for testing
+5. Run ```source ../bin/activate``` to activate the virtualenv
+6. ```export PYTHONPATH=.compiled/```
+7. mypy --strict /home/silver/zubax/yakut/yakut /home/silver/zubax/yakut/tests
+## A manual test
+
+One test can be performed manually, it is located: https://github.com/UAVCAN/yakut/blob/a6ad0fdd9667d894434e00b6c751b2bd7345f684/tests/cmd/monitor.py#L340-L362
+in the `tests/cmd` directory, `monitor.py` file, starting on line 340. It is not automated because 
```

Comment:
Replace this with a reference on where to look for manual tests.

Reply:
Replacing it with itself won't change it